domains/*.shkumaraman.is-local.org.json

```diff
@@ -0,0 +1,14 @@
+{
+    "description": "A SHKUMARAMAN Company",
+    "domain": "is-local.org",
+    "subdomain": "*.shkumaraman",
+    "owner": {
+        "repo": "https://github.com/shkumaraman",
+        "email": "kumarsharma2580@gmail.com"
+    },
+    "record": {
+        "NS1": ["103.76.122.60"],
+        "NS2": ["103.76.122.60"]
+    },
+    "proxied": true
```

Comment:
Invalid records, we do not support NS records.

Reply:
![image](https://github.com/user-attachments/assets/cda4642a-d475-4664-bd55-382d3f6038c3)